pkg/nsx/services/inventory/build.go

```diff
@@ -85,7 +105,7 @@ func (s *InventoryService) BuildPod(pod *corev1.Pod) (retry bool) {
 		ContainerClusterId:      s.NSXConfig.Cluster,
 		ContainerProjectId:      string(namespace.UID),
 		ExternalId:              string(pod.UID),
-		NetworkErrors:           nil,
+		NetworkErrors:           networkErrors,
 		NetworkStatus:           "",
```

Comment:
Should we also update NetworkStatus?
and also need to update compare function

Reply:
fixed